sdcm/cluster_gce.py

```diff
@@ -200,7 +200,7 @@ def _get_persistent_disk_struct(self, name, disk_size, disk_type='pd-ssd', dc_id
                 },
                 "autoDelete": True}
 
-    def add_nodes(self, count, ec2_user_data='', dc_idx=0):
+    def add_nodes(self, count, ec2_user_data='', dc_idx=0, enable_auto_bootstrap=False):
```

Comment:
It will effect `--bootstrap` of ec2 user data, and we set node.
And the default node.enable_auto_bootstrap is false, so we only will update enable_auto_bootstrap of the new added nodes of True.

The node.enable_auto_bootstrap will be used in node.config_setup()